build.yaml

```diff
@@ -805,6 +805,7 @@ steps:
      set -e
      ZONE=$(curl http://metadata.google.internal/computeMetadata/v1/instance/zone -H 'Metadata-Flavor: Google')
      gcloud -q auth activate-service-account --key-file=/test-gsa-key/privateKeyData
+     sleep 60
```

Comment:
What's this for?  This doesn't seem right.

Reply:
There's a timing problem (I think) where test_batch2 finishes and then this delete runs immediately before the worker has had time to shut down and copied the logs to gcs. I was trying to make sure the logs get copied. 60 seconds is probably not enough time and should probably be 90 or 120. I can take this out as well.

Reply:
So I find these timing-based changes potentially unreliable.  Also, it definitely slows the build down.  There is a shutdown notification on instances.  We should have the worker notice the shutdown and upload the log.  Then it is reliable and we don't have additional delays.